js/ui/tag_box.js

```diff
@@ -783,13 +783,16 @@ var TagBox = SelectBox.inherit({
     _loadItem: function() {
         var dataSource = this._getDataSource(),
             filter = dataSource && dataSource.filter(),
+            searchValue = dataSource && dataSource.searchValue(),
             result;
 
         filter && dataSource.filter(this._userFilter || null);
+        searchValue && dataSource.searchValue(null);
```

Comment:
What happens if searchValue will be 0?

Reply:
The searchValue is always a string. It can not be 0, but I have corrected the test to check this behavior